client/src/lib/dependency-injection/buildApolloGraphqlContainer.js

```diff
@@ -33,7 +33,6 @@ const buildApolloGraphqlContainer = (
   /* eslint-disable max-len, no-unused-vars */
   scaffold(key, config, { force }) {
     /*
-     * @todo build out scaffolding
      * I would recommend adding a default mutation update handlers here, instead of in the `register` method.
      * This way query and mutation could be potentially be linked more easily.
      * Config schema could be found in the `GraphqlManager.js` file.
```

Comment:
All of this and the "proof of concept" with the link below are all part of the todo comment. Please remove all of this as well.

Reply:
Updated